src/HttpUpgradeProtocolAccess.h

```diff
@@ -114,7 +114,14 @@ HttpUpgradeProtocolAccess::forApplicable(const ProtocolView &offer, const Visito
         seenApplicable = true; // may already be true
     }
     if (!seenApplicable && other) // OTHER is applicable if named rules were not
-        (void)visitor(ProtoOther, other);
+        (void)visitor(ProtoOther(), other);
+}
+
+const inline SBuf &
```

Comment:
For consistency with the existing code, please put `inline` first:

```suggestion
inline const SBuf &
```

or

```suggestion
inline const auto &
```